Mathlib/Algebra/Opposites.lean

```diff
@@ -49,31 +49,30 @@ universe u v
 
 open Function
 
+/-- Auxiliary type to implement `MulOpposite` and `AddOpposite`.
+
+It turns out to be convenient to have `MulOpposite α= AddOpposite α` true by definition, in the
```

Comment:
```suggestion
It turns out to be convenient to have `MulOpposite α = AddOpposite α` true by definition, in the
```

Reply:
Did you mean to commit this before merging? I guess it doesn't really matter, it can be fixed later.